src/platform/packages/shared/shared-ux/button_toolbar/src/popover/popover.test.tsx

```diff
@@ -31,7 +31,8 @@ describe('<ToolbarPopover />', () => {
     expect(mockHandler).toHaveBeenCalled();
   });
 
-  test('defaults to a bordered empty button', () => {
+  // TODO: fails after adding emotion css prop babel preset at button.prop('css')
```

Comment:
no need to skip this passes:

![image](https://github.com/user-attachments/assets/0cea1a42-fd14-41ac-a994-bfa2a60fd699)

Reply:
great, thank you. 
also plan to add these matches globally in the follow up